shimmingtoolbox/shim/sequencer.py

```diff
@@ -327,8 +327,8 @@ def get_resampled_masks(self):
                 for i in range(n_shims))
 
             # We need to transpose the mask to have the good dimensions
-            resampled_mask = np.array([results_mask[it][0].get_fdata() for it in range(n_shims)]).transpose(1, 2, 3, 0)
-            masks_fmap = np.array([results_mask[it][1].get_fdata() for it in range(n_shims)]).transpose(1, 2, 3, 0)
+            resampled_mask = np.array([results_mask[it][1].get_fdata() for it in range(n_shims)]).transpose(1, 2, 3, 0)
```

Comment:
Can you give more descriptive names to the variables? What is resampled_mask? How is it different from masks_fmap?

Reply:
We can probably go with resampled_mask_dilated or something like that ? 
I think that we should probably go with another PR after this one to fixed all of the confused names in the code !

Reply:
I suggest improving the ones you actively interact with in the PR you are working. I would put a dedicated PR for that as low priority but we can keep it in the back of our minds when we do some changes to improve the codebase.

Reply:
So for this PR, is masks_fmap_dilated, and masks_fmap good for you ?